pkg/asset/machines/master.go

```diff
@@ -537,7 +535,6 @@ func (m *Master) Machines() ([]machineapi.Machine, error) {
 	awsapi.AddToScheme(scheme)
 	azureapi.AddToScheme(scheme)
 	baremetalapi.AddToScheme(scheme)
-	gcpapi.AddToScheme(scheme)
```

Comment:
I think we will need to add the GCP types into the schema here.
e.g. https://github.com/openshift/installer/blob/c3a4dbc69860b9b66b019461ce3fc3c546e1914f/pkg/asset/machines/master.go#L545-L548